### PR TITLE
form can be filled out with the aid of a screen reader and without th…

### PR DESCRIPTION
**Pull request by havok2905**
…e use of a mouse

Hold of on merging this. I want to put this in front of somebody with some accessibility background. This is my first time making use of aria standards and working with accessibility for single page apps.

### CODE REVIEW

File: app/components/tree-form.html [hunk 1]
```diff
@@ -5,80 +5,171 @@
...
+             class='form__label--required'>First Name</label>
+      <input cfo-auto-focus
+             ng-model='$ctrl.firstName'
+             aria-labelledby='firstNameLabel'
+             aria-describedby='firstNameLabelValidationError'
+             name='firstName'
+             type='text'
+             placeholder='First Name'
+             required>
+      <p id'firstNameLabelValidationError'
```
Comment (havok2905) [inline app/components/tree-form.html]: Hey, hey Chris, you forgot something on this line.

File: app/components/tree-form.html [hunk 2]
```diff
@@ -5,80 +5,171 @@
...
+      <label id='addressLabel'
+             for='address'
+             class='form__label--required'>Address</label>
+      <input cfo-auto-focus
+             ng-model='$ctrl.address'
+             aria-labelledby='addressLabel'
+             aria-describedby='addressLabelValidationError'
+             name='address'
+             type='text'
+             placeholder='Adress'
```
Comment (havok2905) [inline app/components/tree-form.html]: Address

File: app/components/tree-form.html (hunk 1 above)
Comment (LondonAtlas) [inline app/components/tree-form.html]: id='firstNameLabelValidationError

**Comment by havok2905** [comment]
I fixed those html errors. :+1:

**Pull request merged**